pandas/core/frame.py

```diff
@@ -7035,6 +7035,12 @@ def value_counts(
                     NaN            1
         dtype: int64
         """
+        warnings.warn(
+            "In pandas 2.0.0, the name of the resulting Series will be "
+            "'count' (or 'proportion' if `normalize=True`).",
+            FutureWarning,
```

Comment:
this is going to be extremely noisy 

is there a way to do so that it doesn't show the warnjng

Reply:
Responding to https://github.com/pandas-dev/pandas/pull/49640#issuecomment-1312560850 as well. 

I thought @jreback expressed the idea of having the warning (with no way to silence) on the call; it seems like I misunderstood. My preferred route would be no warning; another option is to just have a DeprecationWarning - anyone running an interactive session won't see the warning, but those testing via pytest would. While I dislike the idea of adding `name`, I'm not -1 on it.

From the linked issue on adding `name`, I'm taking some liberties here but it seems @jorisvandenbossche is +1, @jbrockmendel is -0.5, @Dr-Irv is +1, and @MarcoGorelli was -1.

Reply:
I presume there will be a 1.5.3 release before 2.0.0? If so, perhaps let's put this off until then, don't want to rush a decision

I wouldn't be keen on adding `name`, but I'd still prefer that to not making the change

Reply:
if we did a DeprecationWarnjng though then this just is really a breaking change and we should just do it

so either am ok with either
- just change it in 2.0
- engineer a nice way to warn in 1.5.x but cannot be so noisy it's everywhere